src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/ListCommand.java

```diff
@@ -150,19 +151,24 @@ private static String toString(HyperlinkedSerializableRecordingDescriptor descri
                 ArrayUtils.addAll(
                         descriptor.getClass().getSuperclass().getDeclaredMethods(),
                         descriptor.getClass().getDeclaredMethods());
+
+        List<String> urls = new ArrayList<String>();
         for (Method m : methods) {
             if (m.getParameterTypes().length == 0
                     && (m.getName().startsWith("get") || m.getName().startsWith("is"))) {
-                sb.append("\t");
-                sb.append(m.getName());
-
-                sb.append("\t\t");
-                sb.append(m.invoke(descriptor));
-
-                sb.append("\n");
+                if (m.getName().contains("Url")) {
```

Comment:
This is a bit fragile, in case the method is renamed from `getDownloadUrl` to `getDownloadURL` for example. Could it be made case-insensitive?

Reply:
Ok, should've fixed it.